packages/apps-electron/src/electron/autoUpdater.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright 2017-2020 @polkadot/apps authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AppUpdater } from 'electron-updater';
+
+export async function setupAutoUpdater (): Promise<void> {
+  const { autoUpdater } = await import('electron-updater');
+
+  await setLogger(autoUpdater);
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
+  autoUpdater.checkForUpdatesAndNotify();
```

Comment:
Adding `await` here should allow us to get rid of `eslint-disable`.

Reply:
But do we want it? I thought it was omitted on purpose so that it doesn't slow down startup

Reply:
You can just do `autoUpdater.checkForUpdatesAndNotify().catch(console.error);` - not  bad idea anyway